doc/languages.rst

```diff
@@ -169,9 +169,15 @@ to view the docstring of the syntax checker.  Likewise, you may use
          A list of include directories.  Relative paths are relative to the file
          being checked.
 
-      .. option:: flycheck-cppcheck-language-standard
+      .. option:: flycheck-cppcheck-standards
 
-         The C or C++ language standard to use via ``--std=``.
+         The C,  C++ and/or  POSIX standards  to use  via one  or more
+         ``--std=``.
+
+      .. option:: flycheck-cppcheck-suppressions
+
+         The  cppcheck  suppressions  list  to use  via  one  or  more
+         ``--suppress=``.
```

Comment:
Hehe... true... I have it set up this way because I write my notes in emacs, and paragraphs in text files look a lot better this way when both the left and right sides are aligned.